data/siggraph-2018/10.1145-3197517.3201336.json

```diff
@@ -51,5 +51,57 @@
         "Misc. comments": "The project page also contains a windows executable if was not able to run on mac",
         "Software language": "C++"
+    },
+    {
+        "Variant name": "Ubuntu 18.04",
+        "Reviewer name": "Julie Digne <julie.digne@liris.cnrs.fr>",
+        "Is master variant (boolean)": true,
+        "Is variant deprecated (boolean)": false,
+        "Title": "Water surface wavelets",
+        "DOI": "10.1145/3197517.3201336",
+        "Year": 2018,
+        "ACM Keywords": [
+            "Physical simulation"
+        ],
+        "Topic {Rendering, Animation and Simulation, Geometry, Images, Virtual Reality, Fabrication}": "Animation",
+        "Co-authors from academia (boolean)": true,
+        "Co-authors from industry (boolean)": true,
+        "ACM Open Access (boolean)": true,
+        "PDF on the authors' webpage / institution (boolean)": true,
+        "PDF URL": "http://pub.ist.ac.at/group_wojtan/projects/2018_Jeschke_WaterSurfaceWavelets/WaterSurfaceWavelets.pdf",
+        "PDF on Arxiv or any openarchive initiatives (boolean)": false,
+        "Arxiv/OAI page URL": "",
+        "Project URL": "http://visualcomputing.ist.ac.at/publications/2018/WSW/",
+        "Code available (boolean)": true,
+        "If code not available, pseudo-code available (boolean)": false,
+        "If pseudo-code, could the paper be trivially implemented? {0..4}": "",
+        "Code URL": "https://github.com/lecopivo/WaterSurfaceWavelets",
+        "Code URL2": "",
+        "MD5 sum (for archives)": "",
+        "git/hg/svn commit hash or revision number": "",
+        "MD5 sum (for archives) URL2": "",
+        "git/hg/svn commit hash or revision number URL2": "69c88a77b1832b17f2e60e762128df465f7076ab",
```

Comment:
Please use field `git/hg/svn commit hash or revision number URL` and not `git/hg/svn commit hash or revision number URL2`
```suggestion
        "git/hg/svn commit hash or revision number": "69c88a77b1832b17f2e60e762128df465f7076ab",
        "MD5 sum (for archives) URL2": "",
        "git/hg/svn commit hash or revision number URL2": "",
```